happypose_ros/happypose_ros/happypose_node.py

```diff
@@ -631,6 +642,26 @@ def _await_results(self, cam_data: dict) -> None:
                     results, header, has_bbox=not self._multiview
                 )
 
+                ##################
+                # for contact_graspnet_ros
+                masks = results["masks"].numpy()
```

Comment:
There is no empty mask published in case no detections were found in the image

Reply:
I don't understand, if there are no detections, an uint16 image full of zeros is published with the current implementation no? Are you saying we should not publish it?